examples/PAV3MultipleCalculation.R

```diff
@@ -209,9 +244,10 @@ main <- function() {
     paste0(host,
            "/analytics/engines/pa/v3/components?document=",
            paDocument)
-  
+  paramlst <- list(document = paDocument)
```

Comment:
Very minor: Can you please rename the variable to `params` or `queryParams` or something similar? It is always preferred to hide the datatypes in variable names.

Reply:
Were you adding `document` query parameter at two places? I think that you can remove it from `componentsLookupUrl`.

Reply:
for retrieving component Id passing paDocument only at this one place